resolwe_bio/processes/support_processors/bam_conversion.py

```diff
@@ -0,0 +1,173 @@
+        "expression-engine": "jinja",
+        "executor": {"docker": {"image": "resolwebio/rnaseq:4.11.0"}},
+        "resources": {"cores": 1, "memory": 4096},
+    }
+    data_name = "Bedtools bamtobed ({{alignment|sample_name|default('?')}})"
+    version = "1.0.0"
+    process_type = "data:bedpe:"
+    category = "Other"
+    entity = {"type": "sample"}
+    scheduling_class = SchedulingClass.BATCH
+
+    class Input:
+        """Input fields."""
+
+        alignment = DataField("alignment:bam", label="Alignment BAM file")
+        species = StringField(
+            label="Species",
+            allow_custom_choice=True,
+            choices=[
+                ("Homo sapiens", "Homo sapiens"),
+                ("Mus musculus", "Mus musculus"),
+                ("Rattus norvegicus", "Rattus norvegicus"),
+                ("Dictyostelium discoideum", "Dictyostelium discoideum"),
+                ("Odocoileus virginianus texanus", "Odocoileus virginianus texanus"),
+                ("Solanum tuberosum", "Solanum tuberosum"),
+            ],
+        )
+        build = StringField(label="Build")
+
+    class Output:
+        """Output fields."""
+
+        bedpe = FileField(label="BEDPE file")
+        species = StringField(label="Species")
+        build = StringField(label="Build")
+
+    def run(self, inputs, outputs):
+        """Run the analysis."""
+        path = inputs.alignment.bam.path
+        basename = os.path.basename(path)
+        assert basename.endswith(".bam")
+        name = basename[:-4]
+        bed_file = f"{name}.bedpe"
+
+        samtools_param = ["-n", path]
+        bedtools_param = ["-bedpe", "-i"]
+
+        (
+            Cmd["samtools"]["sort"][samtools_param]
+            | Cmd["bedtools"]["bamtobed"][bedtools_param]
+            > bed_file
+        )()
+
+        if not os.path.exists(bed_file):
+            self.error("BamToBed process failed.")
+
+        outputs.bedpe = bed_file
+        outputs.species = inputs.species
+        outputs.build = inputs.build
+
+
+class ScaleBigWig(Process):
+    """Creates a scaled bigwig file."""
+
+    slug = "scale-bigwig"
+    name = "bamCoverage"
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {"docker": {"image": "resolwebio/rnaseq:4.11.0"}},
+        "resources": {"cores": 1, "memory": 4096},
+    }
+    data_name = "Scale BigWig ({{alignment|sample_name|default('?')}})"
+    version = "1.0.0"
+    process_type = "data:coverage:bigwig"
+    category = "Other"
+    entity = {"type": "sample", "input": "alignment"}
+    scheduling_class = SchedulingClass.BATCH
+
+    class Input:
+        """Input fields."""
+
+        alignment = DataField("alignment:bam", label="Alignment BAM file")
+        bedpe = DataField(
+            "bedpe",
+            label="BEDPE Normalization factor",
+            description="The BEDPE file describes disjoint genome features, "
+            "such as structural variations or paired-end sequence alignments."
```

Comment:
Add space after the final full stop since these character strings are concatenated.